pro_tes/config.yaml

```diff
@@ -140,3 +140,8 @@ tes:
 
 storeLogs:
   execution_trace: True
+
+middlewares:
+  mutate_destination:
+    - - 'pro_tes.middleware.middleware.DistanceTaskDistribution'
+      - - 'pro_tes.middleware.middleware.RandomTaskDistribution'
```

Comment:
Hmmm, I'm actually not too happy with this structure (and syntax).
I think having a list structure like this would make more sense:

```py
[[mw1, mw1_fb1, mw1_fb2], [mw2, mw2_fb1], mw3]
```

So have one outer list item for each middleware group, where each item can either be a list of options tried in sequential order until the first one passes _or_ a plain string (if there are no fallbacks).

The above in YAML would then become:

```yaml
- - mw1
  - mw1_fb1
  - mw1_fb2
- - mw2
  - mw2_fb1
- mw3
```

OR

```yaml
- [mw1, mw1_fb1, mw1_fb2]
- [mw2, mw2_fb1]
- mw3
```

For this example it would be:

```yaml
middlewares:
   - - 'pro_tes.middleware.middleware.DistanceTaskDistribution'
     - 'pro_tes.middleware.middleware.RandomTaskDistribution'
```